core/domain/feedback_services.py

```diff
@@ -463,7 +677,12 @@ def _get_all_recipient_ids(exploration_id, thread_id, author_id):
 
 
 def _send_batch_emails(recipient_list, feedback_message_reference):
-    """send feedback message email in batch."""
+    """send feedback message email in batch.
```

Comment:
Adds the given FeedbackMessageReference to each of the recipient's email buffers.